src/components/DateRangePickerInputController.jsx

```diff
@@ -123,7 +129,10 @@ export default class DateRangePickerInputController extends React.Component {
       !isInclusivelyBeforeDay(endDate, startDate);
     if (isEndDateValid) {
       onDatesChange({ startDate, endDate });
-      if (!keepOpenOnDateSelect) onFocusChange(null);
+      if (!keepOpenOnDateSelect) {
+        onFocusChange(null);
```

Comment:
Maybe this should call `this.onClearFocus` instead? (To avoid repeating this pair of statements)